content/blog/detect-pii/index.md

```diff
@@ -0,0 +1,219 @@
+
+![How Privy works](privy_flowchart.svg)
+
+### How realistic is this data?
+
+Great care was taken to provide highly realistic PII data providers. For instance, to generate organizations Privy draws from a list of 30,000 names I manually compiled from stock exchange listings and government directories. Other personal information like names and addresses can be obtained from the [Fake Name Generator](https://www.fakenamegenerator.com/) and loaded into Privy. Additionally, Privy implements custom generators for rule based PII data like passports, driver’s licenses, mac addresses, and more.
+
+## Benchmarking existing PII classifiers
+
+Now that we have a [synthetic PII dataset](https://huggingface.co/datasets/beki/privy), let’s run PII detection models on it! I have selected two popular off-the-shelf models to benchmark — [Presidio](https://microsoft.github.io/presidio/) (with a [Spacy NER backend](https://huggingface.co/spacy/en_core_web_lg)) and [Flair](https://huggingface.co/flair/ner-english-large). For fairness, I have filtered the dataset down to the PII types that each chosen model supports, removing unsupported entities. Here is a quick overview of the frameworks used:
+
+[SpaCy](https://github.com/explosion/spaCy) is a popular NLP library with support for Named Entity Recognition.
+
+[Presidio](https://microsoft.github.io/presidio/) is an open-source data protection and anonymization sdk developed by Microsoft. By default, it uses a Spacy NER model behind the scenes along with a set of custom PII detection logic.
+
+[Flair](https://github.com/flairNLP/flair) is an NLP library built on PyTorch, featuring some new embedding techniques (Flair embeddings) and models that achieve state-of-the-art performance on common NER datasets.
+
+### Label Conversions
+
+Since the models surveyed use different labels for their entities, I had to translate PII annotations before passing the protocol trace dataset to them. See the diagram below for how I converted labels from one model architecture to the next.
+
+<details closed>
+<summary>PII label conversions</summary>
+
+![](privy_pii_label_conversions.svg)
+
+</details>
+
+## Off-the-shelf Model Results
+
+### How do we evaluate PII detectors?
+
+Intuitively, we want to compute the accuracy of each model after it has classified PII entities in our synthetic dataset.
+
+<img src="./equations/accuracy.svg" style="width:60%; display: block; margin-left: auto; margin-right: auto;">
```

Comment:
This doesn't render in the deploy preview for some reason. Check how other blog posts do include images

Reply:
Fixed - the issue was that I made the svg with white text on a transparent background, so it was only visible in dark mode